tests/functional/RemoteWebElementTest.php

```diff
@@ -23,6 +23,10 @@ public function testShouldGetText()
         $elementWithTextWithSpaces = $this->driver->findElement(WebDriverBy::id('text-with-spaces'));
 
         $this->assertEquals('Foo bar text', $elementWithSimpleText->getText());
+
+        // TODO: Safari is returning text with newline? Should getText manipulate with text?
```

Comment:
@OndraM from specification it's unclear should getText return either "newline" or not. In specifications it's said "as rendered", so from this perspective, I should be getting "newline" and spaces between, so probably chromedriver and geckodriver misbehave.

https://www.w3.org/TR/webdriver/#get-element-text

Reply:
it's already skipped, but I'll leave a to do comment